### DISCUSSION TITLE
Project boards on mobile version

### DISCUSSION THREAD

**Discussion by WilliamCappelletti** [community/community#18127] [Mobile]
It would be great to access the projects from mobile, to check kanban boards on the go.

**Comment by brianlovin** [comment]
Thanks for the note @WilliamCappelletti — we're actively working on this! Stay tuned for updates in the new few weeks and months :)

**Reply by chiefmikey** [reply]
I would absolutely love this feature, being able to manage projects and add to or update them while mobile would be a game changer and dramatically affect my usage and overall projects workflow. Really happy to see this is actively being developed!!

**Reply by mchelen-gov** [reply]
Same here, especially now that Projects is GA! Glad to hear this is planned 🎉

**Reply by atakanokan** [reply]
> Thanks for the note @WilliamCappelletti — we're actively working on this! Stay tuned for updates in the new few weeks and months :)

Is there an update or an ETA on this?

Also is it possible to track the progress on https://github.com/orgs/github/projects/4247/views/1 ?

**Reply by alejop153** [reply]
One year has passed... :(

**Reply by whiplashcd** [reply]
too bad its been one year has passed

**Reply by Sandalorian** [reply]
In the IOS app you can see the project but can’t really add or move items around. Would be great to interact and manage project items on the go.

**Reply by mfsbo** [reply]
Needed this for roadmap and task list view today so switched from tablet to laptop.

**Comment by Makoaee** [comment]
https://hearthis.at/thabanginno587gmail.com/mr-dukesoul-opening-prayer-one-man-show-jaive-yoyo/7dL/
